src/components/MemberSearchPage/MemberTable.js

```diff
@@ -0,0 +1,143 @@
+import { Col, List, Row } from 'antd';
+import React from 'react';
+import { find } from 'lodash';
+import { ROLES } from 'common/constants';
+import { MemberImage } from 'components/MemberSearchPage/ui';
+import './MemberSearchPage.css';
+
+const userRoleDisplayName = userRole => find(ROLES, { type: userRole }).displayName;
+
+/**
+ * Formats a text value. Returns the same text value
+ * with a portion of the text in bold.
+ * @param {String}    value           Text to be formatted (ex. "xx abc yy")
+ * @param {[String]}  highLightValues Array of highlighted texts
+ * (ex. ["...", "xx <em>abc</em> yy", ".."]
+ * @param index
+ * @return {Object} (ex. <div>xx <b>abc</b> yy</div>
+ */
+const FormatLabel = ({ value, highLightValues, index }) => {
+  if (!highLightValues) {
+    return (
+      <div key={index} className={'format-label'}>
+        {value}
+      </div>
+    );
+  }
+
+  const res = [];
+  highLightValues.forEach(h => {
```

Comment:
Not sure index is correct, here. It should be the index of the forEach loop. I would go with "highLightValues.filter().map()"

Reply:
You are correct about the index. I'm changing it. Your other comment about "highLightValues.filter().map" actually point out a mistake on my part. I'm actually discarding with the if condition the interests that are not a "Hit". Your comment made me realize that there is a mistake in my code.  The "forEach" is justified in this case i think. I just need to add an "else" condition to my "if" statement and keep the one not a "hit" un-formatted.

Reply:
humm... actual on second thought the original index was correct. It is not the index of the forEach i should use. But the original index of the passes to the component.